sample/parcel/package.json

```diff
@@ -16,7 +16,7 @@
     "prop-types": "^15.6.2",
     "react": "^16.5.0",
     "react-dom": "^16.5.0",
-    "react-time-picker": "file:react-time-picker-3.7.2.tgz"
```

Comment:
Sorry, I thought I had excluded this change from my commits!

Reply:
No worries. Next time you can use Yarn, and play with yarn link command - much easier way to test stuff :)